src/ui/components/Notice/styles.scss

```diff
@@ -0,0 +1,97 @@
+@import "~core/css/inc/mixins";
+@import "~core/css/inc/vars";
+@import "~ui/css/vars";
+@import "~photon-colors/colors";
+
+.Notice {
+  border-radius: $border-radius-s;
+  display: grid;
+  font-size: $font-size-s;
+  grid-gap: 4px;
+  grid-template-columns: 20px auto;
+  line-height: 15px;
+  min-height: 20px;
```

Comment:
Same throughout re: `6px` base multiple; these all seem to be multiples of `4` which isn't common in Add-ons or Photon and is in fact a new multiple.